services/workflows-service/src/customer/customer.controller.external.ts

```diff
@@ -37,8 +39,18 @@ export class CustomerControllerExternal {
       };
     }
 
+    const apiKey =
```

Comment:
Please consult with @liorzam around this area, he changed the use of API keys recently, and as far as I remember, `authenticationConfiguration` will be deprecated.

Reply:
It's ok because it still uses this value to generate the new `ApiKey` model record. Thanks